backend/tests/modules/analytics/test_analytics_service.py

```
@@ -0,0 +1,218 @@
+        assert "no recorded activity" in factors
+        assert signals["inactivity"] == 1.0
+        assert signals["not_started"] == 1.0
+
+    def test_healthy_student_scores_low(self, service):
+        now = self._now()
+        instances = [
+            FakeInstance(status="graded", grade=92.0),
+            FakeInstance(status="graded", grade=88.0),
+            FakeInstance(status="submitted"),
+        ]
+        score, factors, signals = service._score_student_risk(
```

Ruff (0.15.15)

[warning] Unpacked variable `factors` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)